src/main_menu.h

```diff
@@ -24,8 +25,14 @@ class main_menu
         std::vector<std::string> vMenuItems; // MOTD, New Game, Load Game, etc.
         std::vector< std::vector<std::string> > vMenuHotkeys; // hotkeys for the vMenuItems
 
+        // Play a sound whenver the user moves left or right in the main menu or its tabs
```

Comment:
It doesn't need to be implemented there. I just have a habit of implementing short functions in the header. I'd be glad to move it.